bus-mapping/src/evm/opcodes/caller.rs

```diff
@@ -12,8 +12,8 @@ use eth_types::GethExecStep;
 pub(crate) struct Caller;
 
 impl Opcode for Caller {
-    fn gen_associated_ops(
-        state: &mut CircuitInputStateRef,
+    fn gen_associated_ops<C: CircuitsParams>(
+        state: &mut CircuitInputStateRef<C>,
```

Comment:
I think it would be great if we can remove this generic from `CircuitInputStateRef`.